### PR TITLE
Hayden

### PR DESCRIPTION
**Pull request by HaydenStegman**
Updated the home page and css.
Added a new component that will feature on the home page the data from a random pet.

### CODE REVIEW

File: src/panels/Home.module.css [hunk 1]
```diff
@@ -85,16 +85,18 @@
 
 .left, .right
 {
-  margin: 0px var(--lr_padding);
```
Comment (Michael12309) [inline src/panels/Home.module.css]: This looks like an error

File: src/panels/Home.module.css [hunk 2]
```diff
@@ -85,16 +85,18 @@
...
+  margin: px var(--lr_padding);
```
Comment (Michael12309) [inline src/panels/Home.module.css]: This looks like an error

Comment (HaydenStegman) [reply src/panels/Home.module.css]: Oh, yea sorry this was an accidental delete.

Comment (Michael12309) [reply src/panels/Home.module.css]: That's fine, it doesn't change anything.  The rest is good, I'm glad you figured everything out

**Pull request merged**